specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2020-07-01-preview/kubernetesconfiguration.json

```diff
@@ -519,6 +529,77 @@
               "description": "Compliance Status of the Configuration",
               "readOnly": true,
               "$ref": "#/definitions/ComplianceStatus"
+            },
+            "creationTime": {
```

Comment:
these two can be moved to the uber systemData top level property that was recently introduced by ARM. Details here - https://armwiki.azurewebsites.net/api_contracts/ResourceSystemData.html?q=systemdata